debian/copy_clang_binaries.sh

```diff
@@ -8,7 +8,7 @@ PATH="/usr/local/bin:/usr/local/sbin:/usr/bin:$PATH"
 LD=$(command -v gold || command -v ld.gold || command -v ld)
 
 if [ -z "$CLANG" ]; then
-    CLANG=$(which clang)
+    CLANG=$(which clang-4.0 || which clang)
```

Comment:
clang 5 (pre release candidate) and clang 6 (current trunk) are also Ok and should be preferred over clang 4.